### DISCUSSION TITLE
Background dim not applying equally on storyboard sprites

### DISCUSSION THREAD

**Discussion by Shmiklak** [ppy/osu#26304] [Q&A]
### Type

Cosmetic

### Bug description

There seems to be some issue with how dim is handled in lazer compared to stable, which I have noticed on this beatmap: https://osu.ppy.sh/beatmapsets/1910029#osu/3939705

This is how it looks in stable:
![image_2023-12-30_23-45-21](https://github.com/ppy/osu/assets/20987315/416b4b46-fa1b-4823-b744-cdd892694c45)

And this is how it looks in lazer:
![image_2023-12-30_23-40-15](https://github.com/ppy/osu/assets/20987315/0f51cbaa-26fc-4670-8144-18c89315e5d9)

You can easily notice the tone difference on the right side, which does not exist in stable version. It does look fine when dim is set to 0, however considering most people would have some level of dim applied this might cause an unwanted look for most of them.

### Version

2023.1229.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13798971/compressed-logs.zip)

**Comment by Coppertine** [comment]
Just to help the peeps out if they want to look into this issue. I find that's the color difference between `sb/convo/mock.png` and `sb/etc/p.png` (1x1 white pixel png). I feel it's how osu!framework handles dimming of 1x1 pixel images upscaled.
